packages/truffle-contract-schema/spec/contract-object.spec.json

```diff
@@ -11,6 +11,11 @@
       "description": "Name used to identify the contract",
       "default": "Contract"
     },
+    "contract_name": {
+      "allOf": [{ "$ref": "#/definitions/ContractName" }],
+      "description": "Name used to identify the contract",
+      "default": "Contract"
+    },
```

Comment:
Blech, can we not do this? (Is it possible?)

Reply:
i believe changing this [line](https://github.com/trufflesuite/truffle/pull/2220/files#diff-1e819f795cdbdcfa8915a5e598b511b2R49) to use `contractName` would be a breaking change 😞 .

Reply:
Shouldn't be breaking. That's what `normalize` is for... `contractName` is the schema-correct version, and `contract_name` should get converted via normalize.